packages/compass-aggregations/src/modules/stage.spec.js

```diff
@@ -310,20 +431,21 @@ describe('Stage module', () => {
       it('generates timestamp', () => {
         expect(generated.ts.low).to.equal(-321469502);
       });
+
       it('returns the stage string', () => {
         expect(generateStageAsString(stage)).to.deep.equal(`{$match: {
-  code: Code('some code'),
-  oid: ObjectId('5a7382114ec1f67ae445f778'),
-  bin: Binary('aakjadfjadfldksjfadf', '1'),
-  dbref: DBRef('db.coll', '1'),
-  nl: NumberLong('3'),
-  nd: NumberDecimal('5.00000001'),
-  ni: NumberInt('5'),
-  minkey: MinKey(),
-  maxkey: MaxKey(),
-  isodate: ISODate('1999-01-01'),
-  regexp: RegExp('/^[a-z0-9_-]{3,16}$/'),
-  ts: Timestamp(-321469502, 367)
+ code: Code('some code'),
+ oid: ObjectId('5a7382114ec1f67ae445f778'),
+ bin: BinData(1, 'YWFramFkZmphZGZsZGtzamZhZGY='),
+ dbref: DBRef('coll', '1'),
```

Comment:
Does this matter?

Reply:
Hm … I think it does? Not sure about how `DBRef` would be used across databases, but the fact there is a database field in them seems to make it clear that if one is provided it should be preserved

Reply:
OK let me see where that's coming from.

Reply:
Here: https://github.com/mongodb-js/query-parser/blob/406ed4ec774b9dea707c39b694e67d8d085545bc/lib/stringify.js#L56-L58

Reply:
And that has no tests..

Reply:
https://github.com/mongodb-js/query-parser/pull/103